### PR TITLE
Feature: Chandlier

### PR DESCRIPTION
**Pull request by hassanmohamed2003**
## Description
Added Chandelier

## Setting up testing environment
1. In the project window navigate to Project/Assets/Scenes and open the "Game 1" scene.
2. Press Play.

#### Chandelier
Criteria:
- [x] Able to drop Chandelier
- [x] NPC starts reacting when the Chandelier hits ground
- [x] Interact message disables after first time

## Checklist
_These checks should always be true for your pull request_
- [x] My pull request is for one story/feature.
- [x] Every individual commit in this pull request is logical.
- [x] All code, documentation, commits and player seen texts are in English.
- [x] I have deleted unused / unnecessary code.
- [x] If my edits need a change in documentation, then I have updated the documentation.
- [x] All code is in correspondence with the code conventions.

### CODE REVIEW

**Review by dcsmit — CHANGES_REQUESTED** [review]
Code looks good but the way drop works should be changed to a different approach as the spring joint is very buggy.

File: Assets/Scripts/Interactable/Interactions/Drop.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using static UnityEditor.Experimental.GraphView.GraphView;
+
+public class Drop : MonoBehaviour
+{
+
+    private HingeJoint _hingeJoint;
+    public GameObject connectedBody;
```
Comment (dcsmit) [inline Assets/Scripts/Interactable/Interactions/Drop.cs]: This variable should be private

**Review by dcsmit — CHANGES_REQUESTED** [review]
Looks good but still needs one change

File: Assets/Scripts/Interactable/Interactions/Drop.cs (hunk 1 above)
Comment (hassanmohamed2003) [reply Assets/Scripts/Interactable/Interactions/Drop.cs]: you are right

**Review by dcsmit — APPROVED** [review]
Looks good! Approved

**Pull request merged**